src/pcapi/core/users/models.py

```diff
@@ -0,0 +1,29 @@
+from datetime import datetime
+
+from sqlalchemy import BigInteger
+from sqlalchemy import Column
+from sqlalchemy import DateTime
+from sqlalchemy import ForeignKey
+from sqlalchemy import String
+from sqlalchemy.orm import relationship
+
+from pcapi.models.db import Model
+from pcapi.models.pc_object import PcObject
+
+
+class Token(PcObject, Model):
+    __tablename__ = "token"
+
+    id = Column(BigInteger, primary_key=True, autoincrement=True)
+
+    userId = Column(BigInteger, ForeignKey("user.id"), index=True, nullable=False)
+
+    user = relationship("UserSQLEntity", foreign_keys=[userId], backref="tokens")
+
+    value = Column(String, index=True, unique=True, nullable=False)
+
+    type = Column(String, nullable=False)
+
+    creationDate = Column(DateTime, nullable=False, default=datetime.utcnow)
```

Comment:
```suggestion
    creationDate = Column(DateTime, nullable=False, server_default=func.now())
```
avec `from sqlalchemy import func`.
Ca permet d'ajouter le default dans la BDD au lieu que ce soit uniquement fait par l'application.

Reply:
ah super ! j'étais étonné de voir que c'était pas généré en contrainte de BDD alors que c'est utilisé partout dans les modèles...